src/containers/Tablet/Tablet.tsx

```diff
@@ -48,8 +48,9 @@ export const Tablet = () => {
         error,
     } = useTypedSelector((state) => state.tablet);
 
-    const {nodeId: queryNodeId, tenantName: queryTenantName} = parseQuery(location);
+    const {TabletId, Overall, Leader, Type} = tablet;
 
+    const {nodeId: queryNodeId, tenantName: queryTenantName} = parseQuery(location);
```

Comment:
Maybe `Type` additionally could be passed via search params so it would be rendered immediately? Now we have placeholder `T` until data is fully fetched.

`Type` from tablet data will be used only if there is no `Type` in search. To Tablet page we can get only from `Tablet` component, so there all params are set